### ISSUE TITLE
Playnite laggy ( virtual monitor issue ? )

### ISSUE THREAD

**Issue by YanouHD** [DuoStream/Duo#149]
Hello !

Duo is THE solution I was looking for ! Thank you. I'm having some issues and not sure about the cause.
Playnite fullscreen mode was smooth with Sunshine only, but with Duo it's all laggy.

The first reason I suspected was the use of a virtual display with RDP, which is not optimized for gaming. When using Sunshine, I always used an HDMI dummy plug to output the display and it was butter smooth.

Is there any option to output an instance to an "hardware" monitor ? (Here the HDMI dummy).

I am not sure if this will fix it, taking the fact that games are smooth, only Playnite causes the problem. But I only tried Kingdom Two Crowns.

Thank you !

**Comment by samkitty** [comment]
Yes, Playnite is very laggy through Duo and I don't think anyone has found a fix for it yet. I personally had to move my entire library into Steam as non-steam games to get around it. Using Steam ROM Manager + Legendary Launcher made the process easier, with Steam Input support and all.

**Comment by OctavioMayer** [comment]
Same issue. It works flawlessly on a Sunshine instance but is really slow on Duo. It seems like it has some difficulty rendering in fullscreen mode; the lag is much more noticeable when streaming in UHD.

I hope it gets resolved, there isn’t really an alternative to Playnite.

**Comment by m33ts4k0z** [comment]
Guys can you upload a video to put the "very laggy" into perspective? Because I cant say that my Playnite is very laggy. It lags very slightly when going into game details but I wouldnt say that it is something that bothers me. Using the Remix fullscreen theme.

**Comment by spikewj** [comment]
This issue might be the duplicate of mine.
#135
